src/components/action-popover/action-popover-item.component.js

```diff
@@ -88,17 +92,22 @@ const MenuItem = ({
 
   const onClick = useCallback(
     (e) => {
-      if (!disabled) {
+      if (!disabled && !isHref && !download) {
         onClickProp();
```

Comment:
This is going to error if `onClick` isn't passed in and this condition is true - I think this is why `onClick` is a required prop

Reply:
You need to add `onClickProp();` into the `else if` section as well, a user could want to have a download item but still have an `onClick` callback as well 

This means `onClick` can stay as a required prop